### PR TITLE
Update Caffeine recipes

### PR DESCRIPTION
**Pull request by neilmartin83**
This modifies the Caffeine recipes so they now download Caffeine from the new GitHub repo. Added code signature verification and tweaked munki recipe as we now download a DMG and not a zip file.

### CODE REVIEW

File: Caffeine/Caffeine.munki.recipe [hunk 1]
```diff
@@ -2,71 +2,47 @@
...
+            <key>catalogs</key>
+            <array>
+                <string>testing</string>
+            </array>
+            <key>description</key>
+            <string>Caffeine is a tiny program that puts an icon in the right side of your menu bar. Click it to prevent your Mac from automatically going to sleep, dimming the screen or starting screen savers. Click it again to go back. Right-click (or ⌘-click) the icon to show the menu.</string>
+            <key>display_name</key>
+            <string>Caffeine</string>
+            <key>minimum_os_version</key>
+            <string>10.5</string>
```
Comment (homebysix) [inline Caffeine/Caffeine.munki.recipe:22]: Minor suggestion: the Intelliscape website says the new version is compatible with Yosemite and up: https://www.intelliscapesolutions.com/apps/caffeine

Comment (neilmartin83) [reply Caffeine/Caffeine.munki.recipe]: Thank you @homebysix - amended my commit. :-)

**Comment by Daz-wallace** [comment]
Bump 😊

**Comment by neilmartin83** [comment]
Bumpity bump :)

**Pull request merged**